PeerioTechnologies/peerio-mobile
# chore(ts): converted channels folder to TS [branch ch18108]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/18108/mobile-typescript-conversion-to-typescript-batch-4-channels

#### Testing instructions  
Verify that creating a channel still works as intended
Verify that channel info (especially member list) still works as intended

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
